server/app/views/admin/questions/QuestionPreviewFragment.html

```diff
@@ -0,0 +1,5 @@
+<div id="sample-question">
+  <div
+    th:replace="~{applicant/ApplicantQuestionFragment :: applicant-question(${question}, ${questionRendererParams}, ${stateAbbreviations})}"
```

Comment:
also seems like ApplicantQuestionFragment should be moved out of applicant, and probably into components along with the questions themselves

Reply:
I'll move ApplicantQuestionFragment and the questions in a follow-up PR.

The questions are already in app/views/questiontypes. Can they stay there?

Reply:
oh yeah that's what I meant. Keep the questions where they are and move ApplicantQuestionFragment to the same place. I'm fine with it being in a follow up PR.